easybuild/easyconfigs/e/EUKulele/EUKulele-1.0.4-foss-2020b.eb

```diff
@@ -0,0 +1,93 @@
+easyblock = 'PythonBundle'
+
+name = 'EUKulele'
+version = '1.0.4'
+
+homepage = 'https://github.com/AlexanderLabWHOI/EUKulele'
+description = """EUKulele is a Python program for taxonomic annotation of microbes in metatranscriptomic and
+ metagenomic samples, with special emphasis on eukaryote discovery."""
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+
+dependencies = [
+    ('Python', '3.8.6'),
+    ('SciPy-bundle', '2020.11'),
+    ('Biopython', '1.78'),
+    ('matplotlib', '3.3.3'),
+    ('PyYAML', '5.3.1'),
+    ('Seaborn', '0.11.1'),
+    ('BLAST+', '2.11.0'),
+    ('BUSCO', '4.1.2'),
```

Comment:
It looks like BUSCO 4.0.6 is needed instead, see https://github.com/AlexanderLabWHOI/EUKulele/issues/14 ...